src/api/routes/services.js

```diff
@@ -72,12 +72,29 @@ const router = express.Router();
  *             schema:
  *               type: array
  *               items:
- *                 $ref: '#/components/schemas/Service'
+ *                 type: object
```

Comment:
There is a lot of redundancy with `Service`. Could we avoid that duplication with [schema composition](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/)? We could factor the `id`, `name` and `description`, and simply diverge on the `terms` property.